### PR TITLE
refactor(website): update the code editor demo page

### PR DESCRIPTION
**Pull request by gdostie**
### Proposed Changes

image

### Acceptance Criteria

-   [ ] The proposed changes are covered by unit tests
-   [ ] The potential breaking changes are clearly identified
-   [ ] [README.md](https://github.com/coveo/plasma/blob/master/README.md) is adjusted to reflect the proposed changes (if relevant)

### CODE REVIEW

File: packages/react/src/components/editor/CodeEditor.tsx [hunk 1]
```diff
@@ -17,18 +17,48 @@ const ReactCodeMirror = loadable(() => import('./CodeMirror'), {
...
+     * A callback function executed when the content of the editor changes
+     *
+     * @param code The content of the editor after the change
+     */
     onChange?: (code: string) => void;
+    /**
+     * A callback function executed when the editor is mounted to the DOM
+     */
     onMount?: (codemirror: Controlled) => void;
-    errorMessage?: string;
```
Comment (sallainCoveo) [inline packages/react/src/components/editor/CodeEditor.tsx:26]: Is it expected? If so, I think this property is still used in the component (unless `ICodeEditorProps` extends another interface that exposed this property :thinking: )

Comment (gdostie) [reply packages/react/src/components/editor/CodeEditor.tsx:26]: Where?

Comment (sallainCoveo) [reply packages/react/src/components/editor/CodeEditor.tsx:26]: [packages/react/src/components/editor/JSONEditor.tsx](https://github.com/coveo/plasma/pull/2566/files#diff-1a09f9ff213de68235d8430d5dd170642ec971e38576b6bdba785268d94b988e) line 69 I believe

Comment (gdostie) [reply packages/react/src/components/editor/CodeEditor.tsx:26]: Is it in the `CodeEditor` component? 😜

Comment (sallainCoveo) [reply packages/react/src/components/editor/CodeEditor.tsx:26]: oh whoops! All rights, "je retourne dans mon trou".

**Review by sallainCoveo — APPROVED** [review]

**Pull request merged**